src/xrpld/rpc/BookChanges.h

```diff
@@ -49,13 +49,13 @@ computeBookChanges(std::shared_ptr<L const> const& lpAccepted)
     std::map<
         std::string,
         std::tuple<
-            STAmount,  // side A volume
-            STAmount,  // side B volume
-            STAmount,  // high rate
-            STAmount,  // low rate
-            STAmount,  // open rate
-            STAmount   // close rate
-            >>
+            STAmount,                 // side A volume
+            STAmount,                 // side B volume
+            STAmount,                 // high rate
+            STAmount,                 // low rate
```

Comment:
optional. just a note that it may be worth changing this tuple to a struct in the future.